### PR TITLE
[CS-2435] - Update merchant names to business

### PR DESCRIPTION
**Pull request by eshinkawa**
### Description

- [X] Completes #2435

### Checklist

- [X] All UI changes have been tested on a small device

### Screenshots

image

image

image

image

### CODE REVIEW

File: cardstack/src/components/TransactionConfirmationSheet/displays/Merchant/RegisterMerchantDisplay.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const RegisterMerchantDisplay = ({
       {merchantInfoDID && (
         <TransactionListItem
           headerText="CREATE THIS MERCHANT"
-          title={merchantInfoDID.name || 'Merchant Name'}
+          title={merchantInfoDID.name || 'Business Name'}
```
Comment (danibonilha) [inline cardstack/src/components/TransactionConfirmationSheet/displays/Merchant/RegisterMerchantDisplay.tsx:48]: Should this default to `Business account` ?

File: src/components/expanded-state/PaymentConfirmationExpandedState.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ const PaymentDetailsItem = ({
             <Container flex={2} />
             <Container flex={8}>
               <Text color="blueText" size="smallest" weight="bold">
-                MERCHANT
+                Business
```
Comment (danibonilha) [inline src/components/expanded-state/PaymentConfirmationExpandedState.tsx]: Maybe we should keep this uppercased ? Or should it even be just account ? idk

**Review by danibonilha — COMMENTED** [review]
I believe in this case @kdowlin review's is more important than mine, so I'll let to him approve, just pointed some things but he can validate if it makes sense or not

File: cardstack/src/components/TransactionConfirmationSheet/displays/Merchant/RegisterMerchantDisplay.tsx (hunk 1 above)
Comment (kdowlin) [reply cardstack/src/components/TransactionConfirmationSheet/displays/Merchant/RegisterMerchantDisplay.tsx:48]: Business Name is fine here.

File: src/components/expanded-state/PaymentConfirmationExpandedState.tsx (hunk 2 above)
Comment (kdowlin) [reply src/components/expanded-state/PaymentConfirmationExpandedState.tsx]: Yes should be all caps

**Review by kdowlin — APPROVED** [review]

**Pull request merged**